nss-evening-cohort-14/michelin-star-restaurant-dashboard-michelin-star-restaurant-dashboard-1
# Delete Ingredients

Added feature to delete ingredients by clicking on delete button

## Description
A logged in user can go to ingredients and click on delete, and the ingredient will be deleted. Only logged in users will have this feature.

## Related Issue
fixes #24 

## Motivation and Context
We want authorized users to be able to log in and delete ingredients that are no longer required.

## How Can This Be Tested?
npm start, log in, click 'Ingredients' on the Navbar. Click a delete button, and the DOM should update. 

## Screenshots (if appropriate):

## Types of changes
